app/tests/test_reporter.py

```diff
@@ -0,0 +1,86 @@
+import os
+import pytest
+
+import yaml
+
+from app.reporter import update_cost_reporter, get_current_and_previous_months_dates
+
+
+@pytest.fixture
+def aws_static_costs():
+    return {"current_month": 700, "previous_month": 800}
+
+
+@pytest.fixture
+def aws_accounts_file():
+    aws_accounts_filename = os.path.join(
+        f"{os.getcwd()}/app/tests", "test_accounts.yaml"
```

Comment:
`os.getcwd()}/app/tests` is where the test is running from; not needed

Reply:
The test is running from the repo directory, when applying `aws_accounts_filename = ""test_accounts.yaml"` it will fail.
Let me know if you meant for something else

Reply:
ok got it thanks

Reply:
done